tools/topology/topology1/platform/common/acp-sdw.m4

```diff
@@ -0,0 +1,25 @@
+divert(-1)
+
+dnl ACP_SDW related macros
+
+dnl ACP_CONFIG(format, mclk, bclk, fsync, tdm, sdw_config_data)
+define(`ACP_SDW_CONFIG',
+`}'
+$1
+)
+dnl ACP_SDW_CONFIG_DATA(type, idx, rate, channel, i2s_tdm_mode)
+#i2s_tdm_mode 1-> tdm mode, 0->i2s mode
```

Comment:
TDM for SoundWire? That makes limited sense.

Reply:
removed copy-paste comments